.github/workflows/release-nightly.yml

```diff
@@ -12,12 +12,44 @@ jobs:
     runs-on: ubuntu-22.04
     name: Nightly
     if: github.repository == 'adap/flower'
+    outputs:
+      name: ${{ steps.release.outputs.name }}
+      version: ${{ steps.release.outputs.version }}
     steps:
       - uses: actions/checkout@v4
       - name: Bootstrap
         uses: ./.github/actions/bootstrap
       - name: Release nightly
+        id: release
         env:
           PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
         run: |
           ./dev/publish-nightly.sh
+          echo "name=$(poetry version | awk {'print $1'})" >> $GITHUB_OUTPUT
+          echo "version=$(poetry version -s)" >> $GITHUB_OUTPUT
+
+  build-docker-images:
+    name: Build nightly images
+    if: github.repository == 'adap/flower'
+    uses: ./.github/workflows/_docker-build.yml
+    needs: release_nightly
+    strategy:
+      fail-fast: false
+      matrix:
+        images: [
+            { repository: "flwr/superlink", file-dir: "src/docker/superlink" }
+          ]
+    with:
+      namespace-repository: ${{ matrix.images.repository }}
+      file-dir: ${{ matrix.images.file-dir }}
+      build-args: |
+        FLWR_VERSION=${{ needs.release_nightly.outputs.version }}
+        FLWR_PACKAGE=${{ needs.release_nightly.outputs.name }}
+        PYTHON_VERSION=3.11
```

Comment:
I haven't got any feedback on the version we want to use therefore I chose version `3.11`.

Reply:
I am unsure if it is preferred to specify the Python and Ubuntu versions explicitly. Alternatively, we can remove it and use the default value in the SuperLink Dockerfile. I would prefer the latter one. Let me know what you think!

Reply:
we agreed to use 3.11 for nightly. i removed the versions here because those are already defined in the dockerfile